resources/views/layouts/navbar.blade.php

```diff
@@ -5,18 +5,16 @@
         <span class="navbar-toggler-icon"></span>
     </button>
     <div class="collapse navbar-collapse" id="app-navbar-collapse">
-        @if (!Auth::user())
-            <ul class="navbar-nav mr-auto">
-                <li class="nav-item {{ strpos( url()->current(), route('home', ['type' => 'all']) ) !== false ? 'active' : '' }}">
-                    <a class="nav-link" href="{{ route('home', ['type' => 'all']) }}">Home</a>
+        <ul class="navbar-nav mr-auto">
+            <li class="nav-item {{ strpos( url()->current(), route('home', ['type' => 'all']) ) !== false ? 'active' : '' }}">
+                <a class="nav-link" href="{{ route('home', ['type' => 'all']) }}">Home</a>
+            </li>
+            @if(!Auth::guest() && !(Auth::user()->user_type == 3))
```

Comment:
For blade files, please use `auth()->*` (e.g. `auth()->guest()`) to avoid possible `Class Not found` errors. 
Just like `url()`, `route()` that are already used.